ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -533,6 +546,139 @@ module Intel = struct
     List.map (find_or_create ~__context) types
 end
 
+module MxGPU = struct
+  let amd_vendor_id = 0x1002
+
+  let ( *** ) = Int64.mul
```

Comment:
Should we move those `Int64` definition on the top level given that they are shared among all the vgpu modules?